config/config.go

```diff
@@ -128,6 +129,46 @@ func (cfg *AuctionTimeouts) validate(errs configErrors) configErrors {
 	return errs
 }
 
+func (data *ExternalCache) validate(errs configErrors) configErrors {
+	if data.Host == "" {
+		if data.Path != "" {
+			// Throw an error because we have a blank Host nad a non-blank Path
```

Comment:
Super Nitpick: Mispelled 'and' as 'nad'.

Reply:
Corrected